invisible_cities/cities/beersheba_test.py

```diff
@@ -149,3 +149,15 @@ def test_beersheba_expandvar(deconvolution_config):
     conf['deconv_params']['psf_fname'] = '$ICDIR/database/test_data/PSF_dst_sum_collapsed.h5'
 
     beersheba(**conf)
+
+
+def test_beersheba_copy_kdst(deconvolution_config, ICDATADIR):
```

Comment:
Do any of the other tests check that if a file doesn't contain the kdst everything runs smoothly?

Reply:
The esmeralda used in the other test does not contain DST group, therefore I don't think it is necessary an extra explicit test